services/QuillLMS/engines/evidence/app/models/evidence/research/gen_ai/prompt_example.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: evidence_research_gen_ai_prompt_examples
+#
+#  id               :bigint           not null, primary key
+#  human_feedback   :text
+#  human_status     :string           not null
+#  student_response :text             not null
+#  created_at       :datetime         not null
+#  updated_at       :datetime         not null
+#  dataset_id       :integer          not null
+#
+module Evidence
+  module Research
+    module GenAI
+      class PromptExample < ApplicationRecord
+        HUMAN_STATUSES = [
+          OPTIMAL = 'optimal',
+          SUB_OPTIMAL = 'sub_optimal'
+        ].freeze
+
+        belongs_to :dataset
+
+        validates :human_status, presence: true
```

Comment:
Likely not worth re-opening a discussion, but this is still a little weird for a name.  Sounds like it's storing the status of a human (alive? tired? hungry?). We might consider something like `human_assigned_status` or `staff_assigned_status`